content/boilerplates/notes/1.1.4.md

```diff
@@ -0,0 +1,42 @@
+## Behavior change
+
+- Changed the default behavior for Pilot to enable fall through routes which allows workloads to access external host using the same port as any
+internal HTTP service. This behavior can be controlled by the `PILOT_ENABLE_FALLTHROUGH_ROUTE` environment variable.
+
+## Bug fixes
+
+- Fixed egress route generation for services of type `ExternalName`.
+
+- Added support for configuring Envoy's idle connection timeout, which prevents running out of
+memory or IP ports over time ([Issue 13355](https://github.com/istio/istio/issues/13355)).
+
+- Fixed a crashing bug in Pilot in failover handling of locality-based load balancing.
+
+- Fixed a crashing bug in Pilot when it was given custom certificate paths.
+
+- Fixed a bug in Pilot where it was ignoring short names used as service entry hosts ([Issue 13436](https://github.com/istio/istio/issues/13436)).
+
+- Added missing `https_protocol_options` to the envoy-metrics-service cluster configuration.
+
+- Fixed a bug in Pilot where it didn't handle https traffic correctly in the fall through route case ([Issue 13386](https://github.com/istio/istio/issues/13386)).
+
+- Fixed a bug in Pilot where it didn't remove endpoints from Envoy after they were removed from Kubernetes ([Issue 13402](https://github.com/istio/istio/issues/13402)).
+
+- Fixed a crashing bug in the node agent ([Issue 13325](https://github.com/istio/istio/issues/13325)).
+
+- Added missing validation to prevent gateway names from containing dots ([Issue 13211](https://github.com/istio/istio/issues/13211)).
+
+- Fixed bug where [`ConsistentHashLB.minimumRingSize`](/docs/reference/config/networking/v1alpha3/destination-rule/#LoadBalancerSettings-ConsistentHashLB)
+was defaulting to 0 instead of the documented 1024 ([Issue 13261](https://github.com/istio/istio/issues/13261)).
+
+## Small enhancements
+
+- Updated to the latest version of the [Kiali](https://www.kiali.io) add-on.
+
+- Updated to the latest version of [Grafana](https://grafana.com).
+
+- Added validation to ensure Citadel is only deployed with a single replica ([Issue 13383](https://github.com/istio/istio/issues/13383)).
```

Comment:
How does this work for upgrade?  I'm asking because we always wanted folks to increase number of replicas before upgrade for each control plane components.

Reply:
@lei-tang I believe you made this change?

Reply:
Multi-replica Citadel is still under development; currently for secret-mount-based Citadel when multiple Citadel are deployed, they have race condition. Per the quote from @jcetkov in istio/istio#13383: "it might be appropriate to modify the helm charts not to allow setting the security replica count via variable, if such case is really not supported and results in hard to trace issues."

Reply:
ok, thanks, i validated we only allow 1 replica now.  should we update this?  https://istio.io/docs/setup/kubernetes/upgrade/steps/  -- `To minimize downtime, please ensure your Istio control plane components and your applications are highly available with multiple replicas` to exclude citadel?

Reply:
Yes, Citadel should be excluded from this. I will create a PR to update it.

Reply:
https://github.com/istio/istio.io/pull/4026 has been created to update this.